include/miopen/miopen.h

```diff
@@ -6582,6 +6583,39 @@ MIOPEN_EXPORT miopenStatus_t miopenBackendInitialize(miopenBackendDescriptor_t d
 // CLOSEOUT BackendAPI DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+
+// GLU APIs
+/** @addtogroup glu
+ *
+ *  @{
+ */
+
+/*! @brief Execute a GLU forward contiguous layer
+ *
+ * @param handle                   MIOpen handle (input)
+ * @param inputDesc                Tensor descriptor for data input tensor x (input)
+ * @param inputSplitDesc           Tensor descriptor for splitted data input tensor x (input)
+ * @param inputFirstHalf           First half input data tensor  (input)
+ * @param inputSecondHalf          Second half input data tensor (input)
+ * @param dim                      Dimensions to split. (input)
+ * @param outputDesc               Tensor descriptor for output data tensor y (input)
```

Comment:
Shouldn't this be output?

Reply:
I have changed comments at [6f6fe652](https://github.com/moreh-dev/MIOpen/pull/6/commits/6f6fe652a6f21e1e315ea829f9d3927e5a4085e2)